test/local_sensitivity/sde.jl

```diff
@@ -0,0 +1,139 @@
+  @show res
+  res
+end
+res_ode_forward = ForwardDiff.gradient(G,p)
+#res_ode_reverse = ReverseDiff.gradient(G,p)
+
+res_ode_trackeru0, res_ode_trackerp = Zygote.gradient((u0,p)->sum(concrete_solve(prob_oop_ode,Tsit5(),u0,p,abstol=abstol,reltol=reltol,saveat=Array(t),sensealg=TrackerAdjoint()).^2.0/2.0),u₀,p)
+
+@test isapprox(res_ode_forward[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-4)
+#@test isapprox(res_ode_reverse[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-4)
+@test isapprox(res_ode_p'[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-4)
+@test isapprox(res_ode_p', res_ode_trackerp, rtol = 1e-4)
+
+# SDE adjoint results (with noise == 0, so should agree with above)
+
+Random.seed!(seed)
+prob_oop_sde = SDEProblem(f_oop_linear,σ_oop_linear,u₀,trange,p)
+sol_oop_sde = solve(prob_oop_sde,RKMil(interpretation=:Stratonovich),dt=1e-4,adaptive=false,save_noise=true)
+res_sde_u0, res_sde_p = adjoint_sensitivities(sol_oop_sde,RKMil(interpretation=:Stratonovich),dg!,t
+ 	,abstol=abstol,reltol=reltol,sensealg=BacksolveAdjoint())
+
+function GSDE(p)
+  Random.seed!(seed)
+  tmp_prob = remake(prob_oop_sde,u0=eltype(p).(prob_oop_sde.u0),p=p,
+                    tspan=eltype(p).(prob_oop_sde.tspan)
+					#,abstol=abstol, reltol=reltol
+					)
+  sol = solve(tmp_prob,RKMil(interpretation=:Stratonovich),dt=tend/10000,adaptive=false,saveat=t)
+  A = convert(Array,sol)
+  res = g(A,p,nothing)
+  @show res
+  res
+end
+res_sde_forward = ForwardDiff.gradient(GSDE,p)
+res_sde_reverse = ReverseDiff.gradient(GSDE,p)
+
+res_sde_trackeru0, res_sde_trackerp = Zygote.gradient((u0,p)->sum(concrete_solve(prob_oop_sde,RKMil(interpretation=:Stratonovich),dt=tend/1400,adaptive=false,u0,p,saveat=Array(t),sensealg=TrackerAdjoint()).^2.0/2.0),u₀,p)
+
+
+@test isapprox(res_sde_forward[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-2)
+@test isapprox(res_sde_reverse[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-2)
+@test isapprox(res_sde_p'[1], sum(@. u₀^2*exp(2*p[1]*t)*t), rtol = 1e-2)
+@test isapprox(res_sde_p'[1], res_sde_trackerp[1], rtol = 1e-2)
+
+
+# SDE adjoint results (with noise != 0)
+
+p2 = [1.01,0.87]
+
+Random.seed!(seed)
+prob_oop_sde2 = SDEProblem(f_oop_linear,σ_oop_linear,u₀,trange,p2)
+sol_oop_sde2 = solve(prob_oop_sde2,RKMil(interpretation=:Stratonovich),dt=tend/1000,adaptive=false,save_noise=true)
+res_sde_u02, res_sde_p2 = adjoint_sensitivities(sol_oop_sde2,RKMil(interpretation=:Stratonovich),dg!,t
+ 	,abstol=abstol,reltol=reltol,sensealg=BacksolveAdjoint())
+
+function GSDE(p)
+  Random.seed!(seed)
+  tmp_prob = remake(prob_oop_sde2,u0=eltype(p2).(prob_oop_sde2.u0),p=p,
+                    tspan=eltype(p).(prob_oop_sde2.tspan)
+					#,abstol=abstol, reltol=reltol
+					)
+  sol = solve(tmp_prob,RKMil(interpretation=:Stratonovich),dt=tend/1000,adaptive=false,saveat=t)
+  A = convert(Array,sol)
+  res = g(A,p,nothing)
+  @show res
+  res
+end
+res_sde_forward2 = ForwardDiff.gradient(GSDE,p2)
+res_sde_reverse2 = ReverseDiff.gradient(GSDE,p2)
```

Comment:
I'd just remove ReverseDiff from here

Reply:
Although the tests are passing now for reverse diff?

Reply:
oh cool, nevermind then